chat-plugins/exiled.js

```diff
@@ -832,6 +832,46 @@ exports.commands = {
 			});
 		});
 	},
+	yt: 'youtube',
+	youtube: function (target, room, user) {
+		if (!this.runBroadcast()) return false;
+		if (!target) return false;
+		let params_spl = target.split(' '), g = ' ';
+		for (let i = 0; i < params_spl.length; i++) {
+			g += '+' + params_spl[i];
+		}
+		g = g.substr(1);
+
+		let reqOpts = {
+			hostname: 'www.googleapis.com',
+			method: 'GET',
+			path: '/youtube/v3/search?part=snippet&q=' + g + '&type=video&key=AIzaSyA4fgl5OuqrgLE1B7v8IWYr3rdpTGkTmns',
+			headers: {
+				'Content-Type': 'application/json',
+			},
+		};
+
+		let self = this;
+		let data = '';
+		let req = require('https').request(reqOpts, function (res) {
+			res.on('data', function (chunk) {
+				data += chunk;
+			});
+			res.on('end', function (chunk) {
+				let d = JSON.parse(data);
+				if (d.pageInfo.totalResults === 0) {
+					room.add('No videos found');
+					room.update();
+					return false;
+				}
+				let id = getLinkId(target);
```

Comment:
Define this, actually was this even tested? Its pretty obvious it's not defined and it wouldn't work.